lib/digsync_web/router.ex

```diff
@@ -33,6 +33,9 @@ defmodule DigsyncWeb.Router do
       live("/users", UsersLive)
       live("/users/:user_id", UserDetailsLive)
       live("/create-message/:user_id", PrivateMessageLive)
+      live("/groups", GroupsLive)
+      live("/groups/:group_id", GroupDetailsLive)
+      live("/create-group/:user_id", CreateGroupLive)
```

Comment:
we probably don't need user_id for create group, as the actor is implied. I also don't see @user_id being used in the template? I could be wrong though.